src/pages/Member/SignUpForm/SignUpForm.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
+import './SignUpForm.scss';
+
+const SignUpForm = () => {
+  const [inputState, setInputState] = useState({
+    email: '',
+    password: '',
+    name: '',
+    address: '',
+    phoneNumber: '',
+  });
+
+  const [radioGroup, setRadioGroup] = useState({
+    isFamily: false,
+    isNotFamily: false,
+  });
+
+  const changeRadio = event => {
+    let obj = {};
+    obj[event.target.value] = event.target.checked;
+    console.log(obj);
+    setRadioGroup(obj);
+  };
+
+  const changeInfo = e => {
+    const { name, value } = e.target;
+    setInputState({
+      ...inputState,
+      [name]: value,
+    });
+  };
+
+  const signUp = () => {
+    fetch('http://10.58.1.126:8000/users/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: inputState.email,
+        password: inputState.password,
+        name: inputState.name,
+        //address: inputState.address,
+        phone_number: inputState.phoneNumber,
```

Comment:
```suggestion
        ...inputState
```
key, value가 동일하다면 단축속성명 및 spread 연산자 조합해서 위와 같이 쓸 수 있겠네요!